app/src/main/java/com/kaajjo/libresudoku/ui/theme/SudokuBoardTheme.kt

```diff
@@ -37,11 +37,11 @@ object BoardColors {
 
     inline val thickLineColor: Color
         @Composable
-        get() = MaterialTheme.colorScheme.onSurfaceVariant.copy(0.65f)
+        get() = MaterialTheme.colorScheme.onSurfaceVariant.copy(0.55f)
 
     inline val thinLineColor: Color
         @Composable
-        get() = MaterialTheme.colorScheme.onSurfaceVariant.copy(0.4f)
+        get() = MaterialTheme.colorScheme.onSurfaceVariant.copy(0.25f)
```

Comment:
I think we should also change them in non-monet colors https://github.com/kaajjo/Libre-Sudoku/blob/e5e074b5902e5306d594de843cccaf391463110b/app/src/main/java/com/kaajjo/libresudoku/MainActivity.kt#L147-L148